common/views/PolicySection.jsx

```diff
@@ -82,35 +80,19 @@ class PolicySection extends React.Component {
         let contentAndRate = (
           <div>
             <h3>{commitment.content}</h3>
-            <p style={{color: Colors.darkBlack}}>{totalRateCount} 人評進度</p>
+            <p style={{color: pptColors.darkGray}}>{totalRateCount} 人評進度</p>
           </div>
         );
 
-        //move Progressicon to here.
-        let progressIcon;
-        if (progress === 'done') {
-          progressIcon = (
-            <DoneIcon style={{fill: 'green'}}></DoneIcon>
-          );
-        }else if (progress === 'doing') {
-          progressIcon = (
-            <DoingIcon style={{fill: 'yellow'}}></DoingIcon>
-          );
-        }else {
-          progressIcon = (
-            <NotyetIcon style={{fill: 'red'}}></NotyetIcon>
-          );
-        }
-
         return (
             <ListItem
-              leftIcon={progressIcon}
+              leftIcon={progressIconPicker(progress)}
```

Comment:
On first thought, I wanna name it as determiner, but I thought the name was too long